src/azure-cli/azure/cli/command_modules/monitor/operations/autoscale_settings.py

```diff
@@ -72,8 +72,7 @@ def autoscale_update(instance, count=None, min_count=None, max_count=None, tags=
     if enabled is not None:
         instance.enabled = enabled
 
-    if any([count, min_count, max_count]):
-
+    if count is not None or min_count is not None or max_count is not None:
```

Comment:
`count == 0` now is `true` (previous it is `false`)?

Reply:
Yes

Reply:
When count == 0 any([count, min_count, max_count]) will response false